### DISCUSSION TITLE
We REALLY need a downvote button

### DISCUSSION THREAD

**Discussion by JCKodel** [dailydotdev/daily#904] [Feature Requests]
Some posts are just plain wrong and nocive, with a lot of wrong assumptions, low-quality opinions or just plain dangerous, teaching people how to do things wrongly.

The voting system should be like Reddit or Stack Overflow: a score, starting with 1, and an increase and a decrease button.

Otherwise, it's easier to uninstall the addon and just search Reddit instead. Tired of seeing shit posts, unrelated posts (such as "This is the order of some move franchise").

**Comment by idoshamun** [comment]
Hi @JCKodel,

You're most definitely right. We already have an open discussion #460.
Hope we can make it live soon 🙏
